### PR TITLE
Include the fwlink in the warning.

### PR DESCRIPTION
**Pull request by huanwu**
@shmao @zhenlan @mconnew

### CODE REVIEW

File: src/Microsoft.XmlSerializer.Generator/pkg/build/Microsoft.XmlSerializer.Generator.targets [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <SerializerDllImmediatePath>$(IntermediateOutputPath)$(SerializerName).dll</SerializerDllImmediatePath>
     <SerializerPdbImmediatePath>$(IntermediateOutputPath)$(SerializerName).pdb</SerializerPdbImmediatePath>
     <SerializerCsImmediatePath>$(IntermediateOutputPath)$(SerializerName).cs</SerializerCsImmediatePath>
-    <SGenWarningText>SGEN : warning SGEN1: Fail to generate the serializer for $(AssemblyName).dll. Please check the build log for the details.</SGenWarningText>
+    <SGenWarningText>SGEN : warning SGEN1: Fail to generate the serializer for $(AssemblyName).dll. Please follow the instruction in https://go.microsoft.com/fwlink/?linkid=858594 and try again.</SGenWarningText>
```
Comment (stephentoub) [inline src/Microsoft.XmlSerializer.Generator/pkg/build/Microsoft.XmlSerializer.Generator.targets]: Nit: instruction => instructions

And right now the fwlink just goes to the corefx repo.  I assume it'll be updated subsequently to point to the right place?

Comment (huanwu) [reply src/Microsoft.XmlSerializer.Generator/pkg/build/Microsoft.XmlSerializer.Generator.targets]: @stephentoub , yeah, we will update it. :)

**Comment by huanwu** [comment]
@dotnet-bot test OSX x64 Debug Build

**Comment by huanwu** [comment]
@dotnet-bot test Linux x64 Release Build

**Pull request merged**